### PR TITLE
biointerchange 2.0.2

### PR DESCRIPTION
**Pull request by indiedotkim**
Fixed a memory leak. License update.

### CODE REVIEW

File: biointerchange.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   homepage "https://www.codamono.com"
   # tag "bioinformatics"
 
   if OS.mac?
-    version "2.0.1+74"
-    url "https://www.codamono.com/brew/osx-10-10/biointerchange-2.0.1+74.tar.gz"
-    sha256 "ed1864b13aaf23518d32867683bfcc9bc8c15902c12b549433832e83b43926a6"
+    version "2.0.2+83"
+    url "https://www.codamono.com/brew/osx-10-10/biointerchange-2.0.2+83.tar.gz"
+    sha256 "fb70c5d537c08cf860a270a156cc1c00e13ec095"
```
Comment (dpo) [inline biointerchange.rb]: This isn't the `sha256`.

**Comment by indiedotkim** [comment]
As @dpo pointed out: sha256 is wrong. My bad! Build script is still using sha1! Going to fix it...

**Comment by indiedotkim** [comment]
SHA's fixed. Commit squashed.

**Comment by dpo** [comment]
Looking good. Thanks!

**Pull request closed without merging**